src/styles/main.scss

```
@@ -1,3 +1,71 @@
+$day-padding: 10px;
+$day-border: 1px;
+$day-gap: 1px;
+$day-size: 100px;
+$start-week:  
+  "mon" 0,
+  "tue" ($day-size + $day_gap),
+  "wed" (2 *($day-size + $day_gap)),
+  "thu" (3 *($day-size + $day_gap)),
+  "fri" (4 *($day-size + $day_gap)),
+  "sat" (5 *($day-size + $day_gap)),
+  "sun" (6 *($day-size + $day_gap));
```

you can calculate it in @each

```suggestion
$start-week:  
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```